src/components/AudioAnalysis/SpAnalysisLayout.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Card,
+  Grid,
+  Typography,
+  CardContent,
+  LinearProgress,
+} from '@material-ui/core';
+
+import Chart from './SpAnalysisChart';
+import SectionChart from './SpAnalysisSectionChart';
+import ChartSize from './SpAnalysisChartSize';
+import ChordButtons from './SpAnalysisChordButtons';
+import PitchSelect from './SpAnalysisPitchSelect';
+import useStore from '../../store/useStore';
+import { fixAnalysis } from '../../utils/spotifyProxies';
+
+export default function SpAnalysisLayout() {
+  const analysis = useStore(
+    (state) => state?.spotify?.spotifyData?.trackAnalysis
+  );
+
+  const [width, setWidth] = useState(2000);
+  const [height] = useState(255);
+  const [pitches, setPitches] = useState({
+    C: true,
+    'C#': true,
+    D: true,
+    'D#': true,
+    E: true,
+    F: true,
+    'F#': true,
+    G: true,
+    'G#': true,
+    A: true,
+    'A#': true,
+    B: true,
+  });
+  const [newAnalysis, setNewAnalysis] = useState({});
+
+  useEffect(() => {
+    const newData = fixAnalysis(analysis);
+    setNewAnalysis(newData);
+  }, [analysis]);
+
+  const handleSizeSliderChange = (event: any, newValue: any) => {
+    setWidth(newValue);
+  };
+
+  const handleCheck = (e: any) => {
+    setPitches((prev) => ({
+      ...prev,
+      [e.target.name]: e.target.checked,
+    }));
+  };
+
+  const handleChordClick = (e: any) => {
+    const pitchClasses = [
+      'C',
+      'C#',
+      'D',
+      'D#',
+      'E',
+      'F',
+      'F#',
+      'G',
+      'G#',
+      'A',
+      'A#',
+      'B',
+    ];
+    const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+    numbers.map((item) => {
+      setPitches((prev) => ({
+        ...prev,
+        [pitchClasses[item]]: false,
+      }));
+      return null;
+    });
+
+    setTimeout(() => {
+      e.target.value.map((item: any) => {
+        setPitches((prev) => ({ ...prev, [pitchClasses[item]]: true }));
+        return null;
+      });
+    });
+  };
+
+  const { segments, sections } = analysis;
+  // console.log({ analysis });
+
+  return (
+    <Card style={{ color: 'white' }}>
```

Comment:
Add collapsalbe accordion, a toggle button or similar to hide it on the first view,
so its not that of an overload for the user